js/source/raster_tile_source.js

```diff
@@ -48,11 +43,9 @@ RasterTileSource.prototype = util.inherit(Evented, {
         };
     },
 
-    getVisibleCoordinates: Source._getVisibleCoordinates,
-    getTile: Source._getTile,
-
-    _loadTile: function(tile) {
+    load: function(tile, cb) {
```

Comment:
We usually write out the whole word `callback`. Avoid acronyms helps the codebase remain readable and approachable for everyone.